tests/test_cli.py

```diff
@@ -1,8 +1,14 @@
+import os
+
 import click
 from click.testing import CliRunner
 
 from solosis.cli import cli
 
+# responding to error message ("missing environmnet variables LSB_DEFAULT_USERGROUP and TEAM_DATA_DIR")
+os.environ["LSB_DEFAULT_USERGROUP"] = "team298"
+os.environ["TEAM_DATA_DIR"] = "/lustre/scratch126/cellgen/team298/data"
```

Comment:
I would suggest, for the sake of testing, you define a dummy location, that can actually be created. For example:
`/tmp/solosis`